### PR TITLE
Add the initial examples from the core data model

### PR DESCRIPTION
**Pull request by OR13**
This PR removes a cross origin script and improves the respec plugin used to process spec examples

It is editorial.

It also changes the initial examples to align with the core data model.

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/vc-jose-cose/pull/162.html" title="Last updated on Oct 13, 2023, 6:29 PM UTC (1f42458)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/vc-jose-cose/162/5fb55fe...1f42458.html" title="Last updated on Oct 13, 2023, 6:29 PM UTC (1f42458)">Diff</a>

### CODE REVIEW

**Review by decentralgabe — APPROVED** [review]

File: index.html [hunk 1]
```diff
@@ -244,26 +244,37 @@ <h2>Securing JSON-LD Verifiable Credentials with JOSE</h2>
...
+      "name": "Example University"
     }
+  },
+  "proof": {
+    "type": "DataIntegrityProof",
+    "cryptosuite": "eddsa-2022",
+    "created": "2023-06-18T21:19:10Z",
+    "proofPurpose": "assertionMethod",
+    "verificationMethod": "https://university.example/issuers/565049#key-123",
+    "proofValue": "zQeVbY4oey5q2M3XKaxup3tmzN4DRFTLVqpLMweBrSxMY2xHX5XTYV8nQApmEcqaqA3Q1gVHMrXFkXJeV6doDwLWx"
```
Comment (selfissued) [inline index.html:271]: @OR13 please remove the `proof` section, as it uses a different securing mechanism than the one we're specifying.
```suggestion
```

File: index.html [hunk 2]
```diff
@@ -284,39 +295,54 @@ <h2>Securing JSON-LD Verifiable Presentations with JOSE</h2>
...
+        "name": "Example University"
+      }
+    },
+    "proof": {
+      "type": "DataIntegrityProof",
+      "cryptosuite": "eddsa-2022",
+      "created": "2023-06-18T21:19:10Z",
+      "proofPurpose": "assertionMethod",
+      "verificationMethod": "https://university.example/issuers/565049#key-1",
+      "proofValue": "zQeVbY4oey5q2M3XKaxup3tmzN4DRFTLVqpLMweBrSxMY2xHX5XTYV8nQApmEcqaqA3Q1gVHMrXFkXJeV6doDwLWx"
```
Comment (selfissued) [inline index.html:328]: Please delete this `proof` for the same reason as the previous.
```suggestion
```

File: index.html [hunk 3]
```diff
@@ -284,39 +295,54 @@ <h2>Securing JSON-LD Verifiable Presentations with JOSE</h2>
...
+  }],
+  "proof": {
+    "type": "DataIntegrityProof",
+    "cryptosuite": "eddsa-2022",
+    "created": "2018-09-14T21:19:10Z",
+    "proofPurpose": "authentication",
+    "verificationMethod": "did:example:ebfeb1f712ebc6f1c276e12ec21#keys-1",
+    "challenge": "1f44d55f-f161-4938-a659-f8026467f126",
+    "domain": "4jt78h47fh47",
+    "proofValue": "zqpLMweBrSxMY2xHX5XTYV8nQAJeV6doDwLWxQeVbY4oey5q2pmEcqaqA3Q1gVHMrXFkXM3XKaxup3tmzN4DRFTLV"
```
Comment (selfissued) [inline index.html:339]: And likewise please delete this `proof`.
```suggestion
  }]
```

**Review by selfissued — CHANGES_REQUESTED** [review]
Please delete the `proof` clauses, per my individual review comments.

File: index.html [hunk 4]
```diff
@@ -244,26 +244,37 @@ <h2>Securing JSON-LD Verifiable Credentials with JOSE</h2>
...
+    "cryptosuite": "eddsa-2022",
+    "created": "2023-06-18T21:19:10Z",
+    "proofPurpose": "assertionMethod",
+    "verificationMethod": "https://university.example/issuers/565049#key-123",
+    "proofValue": "zQeVbY4oey5q2M3XKaxup3tmzN4DRFTLVqpLMweBrSxMY2xHX5XTYV8nQApmEcqaqA3Q1gVHMrXFkXJeV6doDwLWx"
   }
 }
         </pre>
+        <p>
+          See <a data-cite="VC-DATA-MODEL-2.0#example-a-simple-example-of-a-verifiable-credential"></a> for more details regarding this example.
```
Comment (OR13) [inline index.html:276]: @selfissued I can't apply your suggestion, these examples are intentionally verbatim from the core specification.

File: index.html [hunk 5]
```diff
@@ -284,39 +295,54 @@ <h2>Securing JSON-LD Verifiable Presentations with JOSE</h2>
...
+    "verificationMethod": "did:example:ebfeb1f712ebc6f1c276e12ec21#keys-1",
+    "challenge": "1f44d55f-f161-4938-a659-f8026467f126",
+    "domain": "4jt78h47fh47",
+    "proofValue": "zqpLMweBrSxMY2xHX5XTYV8nQAJeV6doDwLWxQeVbY4oey5q2pmEcqaqA3Q1gVHMrXFkXM3XKaxup3tmzN4DRFTLV"
   }
 }
-        </pre>
+      </pre>
+      <p>
+        See <a data-cite="VC-DATA-MODEL-2.0#example-a-simple-example-of-a-verifiable-presentation"></a> for more details regarding this example.
```
Comment (OR13) [inline index.html:344]: @selfissued I can't apply your suggestion, these examples are intentionally verbatim from the core specification.

**Review by decentralgabe — CHANGES_REQUESTED** [review]
agree with @selfissued 's comments

**Review by awoie — CHANGES_REQUESTED** [review]
I don't think the examples should have a `proof` property. That is confusing to me.

**Comment by OR13** [comment]
https://github.com/w3c/vc-data-model/pull/1308

has been merged, I am requesting re-reviews.

**Review by selfissued — APPROVED** [review]
Now that we've merged https://github.com/w3c/vc-data-model/pull/1308 and the corresponding changes have been made here, we're good to go.

**Pull request merged**